### DISCUSSION TITLE
List/array of strings

### DISCUSSION THREAD

**Discussion by CrashServer** [cables-gl/cables_docs#684] [Answers & Questions]
Hi, I might be a very noobish question, but i'm struggling to achieve this :
- create a empty array of strings (10 elements for example) 
- on a trigger change the content at index 0 with an input
- shift all other strings by one index
- not grow the amount of total strings just change the content 
I can do it with ints but can't seem to find a way to do it with strings. 

Thanks in advance !

**Comment by pandrr** [comment]
can you show your int version ? what ops are you missing for strings ?
